test/suites/integration/extended_resources_test.go

```diff
@@ -208,6 +208,48 @@ var _ = Describe("Extended Resources", func() {
 		env.ExpectCreatedNodeCount("==", 1)
 		env.EventuallyExpectInitializedNodeCount("==", 1)
 	})
+
+	It("should provision nodes for a deployment that requests vpc.amazonaws.com/efa", func() {
+		ExpectEFADevicePluginCreated()
+
+		nodePool.Spec.Template.Labels = map[string]string{
+			"aws.amazon.com/efa": "true",
+		}
+		nodePool.Spec.Template.Spec.Taints = []v1.Taint{
+			{
+				Key:    "aws.amazon.com/efa",
+				Effect: v1.TaintEffectNoSchedule,
+			},
+		}
+		numPods := 1
+		dep := test.Deployment(test.DeploymentOptions{
+			Replicas: int32(numPods),
+			PodOptions: test.PodOptions{
+				ObjectMeta: metav1.ObjectMeta{
+					Labels: map[string]string{"app": "efa-app"},
+				},
+				Tolerations: []v1.Toleration{
+					{
+						Key:      "aws.amazon.com/efa",
+						Operator: v1.TolerationOpExists,
+					},
+				},
+				ResourceRequirements: v1.ResourceRequirements{
+					Requests: v1.ResourceList{
```

Comment:
Any to reason to have a test that requests more than one EFA? Are there any edge-cases that we should make sure that we test when we request multiple EFAs which would launch multiple network interfaces?

Reply:
Good call, there should be one to verify that multiple network card launch templates are created correctly.